virttest/utils_misc.py

```diff
@@ -3062,7 +3120,12 @@ def get_image_info(image_file):
                            'csize' : '65536'}
     """
     try:
+        avoid_qemu_lock_param = " -U"
```

Comment:
the variable never changed and used only once, so maybe remove it.

Reply:
This will make it more clear about what's "-U" for in code, but no need to check qemu manual

Reply:
well, a comment will do this, too